src/rpc/blockchain.cpp

```diff
@@ -2388,6 +2487,7 @@ static const CRPCCommand commands[] =
     { "blockchain",         "getbestblockhash",       &getbestblockhash,       {} },
     { "blockchain",         "getblockcount",          &getblockcount,          {} },
     { "blockchain",         "getblock",               &getblock,               {"blockhash","verbosity|verbose"} },
+    { "blockchain",         "getblockbyheight",       &getblockbyheight,       {"blockheight","verbosity|verbose"} },
```

Comment:
Do we need to support both named args like getblock? I don't really know the story on this.

Reply:
Same, the reason why the RPC call is very similar to getblock is becuase I have forked it. That's why I also removed some of the legacy getblock stuff